lib/web_ui/test/canvaskit/canvas_golden_test.dart

```diff
@@ -357,6 +357,27 @@ void testMain() {
       await testTextStyle('height', height: 2);
     });
 
+    test('text styles - leading distribution', () async {
+      await testTextStyle('half leading', height: 20, fontSize: 10, leadingDistribution: ui.TextLeadingDistribution.even);
+      await testTextStyle(
+        'half leading inherited from paragraph',
+        height: 20,
+        fontSize: 10,
+        paragraphTextHeightBehavior: ui.TextHeightBehavior(
+          leadingDistribution: ui.TextLeadingDistribution.even,
+        ),
+      );
+      await testTextStyle(
+        'half leading',
```

Comment:
The first argument is used as part of the golden screenshot file name, so it should be distinct from previous tests.

Reply:
Ops forgot to push. Fixed.